libs/hmpb/layout/src/layout.ts

```diff
@@ -882,7 +882,10 @@ function CandidateContest({
   const options: Rectangle[] = [];
   let rowHeightUsed = headingRowHeight;
   for (const candidate of contest.candidates) {
-    const partyText = getCandidatePartiesDescription(election, candidate);
+    const partyText =
+      election.type === 'primary'
+        ? undefined
+        : getCandidatePartiesDescription(election, candidate);
```

Comment:
I'm noticing that the spacing between the last option and the box border when there are no parties is getting to be a little small:

<img width="178" alt="spacing" src="https://github.com/votingworks/vxsuite/assets/12616928/5ae54416-2e1a-4d8b-941d-317acb59dfd8">

Could modify the styles to add some space back for this case

Reply:
Yeah, good catch. I've looked into this before and you have to add a whole timing mark row space to make the bubble alignment for the subsequent contest work out, which also looks weird. So I just left it as is for now until we remove the bubble-on-grid-intersection constraint.

Reply:
Ah right the timing-mark constraint makes sense - leaving as is is fine then!